ocis-phoenix/pkg/service/v0/service.go

```diff
@@ -70,14 +70,14 @@ func (p Phoenix) getPayload() (payload []byte, err error) {
 
 		if p.config.Phoenix.Config.ExternalApps == nil {
 			p.config.Phoenix.Config.ExternalApps = []config.ExternalApp{
-				{
-					ID:   "accounts",
-					Path: "/accounts.js",
-				},
 				{
 					ID:   "settings",
 					Path: "/settings.js",
 				},
+				{
+					ID:   "accounts",
+					Path: "/accounts.js",
+				},
```

Comment:
I changed the order because it is the order in which the navItems are rendered in the respective menu. We want to settings navItem first after the Profile navItem.